src/bin/psql/common.c

```diff
@@ -790,6 +790,8 @@ StoreQueryTuple(const PGresult *result)
 			{
 				pg_log_warning("attempt to \\gset into specially treated variable \"%s\" ignored",
 							   varname);
+
+				free(varname);
```

Comment:
I don't think we should patch upstream like this.

Reply:
Eah, I was on the fence if to do it or not and wanted to see opinions. I'll replace it with the suppression then